target/linux/imx6/modules.mk

```diff
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: GPL-2.0-only
+#
+# Copyright (C) 2019 OpenWrt.org
```

Comment:
You are not OpenWrt, so you can't add this Copyright ...

Reply:
Removed.